index.html

```diff
@@ -43,11 +43,108 @@ <h1 class="prof">I am Software Developer</h1>
             <a href="#" class="down-button">
                 <img src="./image/Disabled.svg" alt="">
             </a>
-        
-
+        </div>
+    </section>
 
+     <!-- content 2 -->
+    <section class="main2">
+        <div class="title">
+            <h1>My Recent Work</h1>
+            <div class="end-line"></div>
+        </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul>
+                    <li>HTML5</li>
+                    <li>HTML5</li>
+                    <li>HTML5</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
+        </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul >
+                    <li>Ruby on Rails</li>
+                    <li>HTML5</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
         </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul>
+                    <li>Ruby on Rails</li>
+                    <li>HTML5</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
+        </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul>
+                    <li>Ruby on Rails</li>
+                    <li>HTML5</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
+        </div>
+    </section>
+    <!-- content 2 -->
+    <section class="main3">
+        <h1>About me</h1>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+        see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+        <button type="button">Get my Resume</button>
     </section>
-  
+
+<section class="card-info">
+    <div class="card">
```

Comment:
- [ ] let's make sure we use `ul` instead of `section` and `li` instead of `div` tag for Semantic HTML  that would make our code more understandable you could read more about [Semantic HTML](https://developer.mozilla.org/en-US/docs/Glossary/Semantics)

Reply:
Done!